alphadia/workflow/peptidecentric.py

```diff
@@ -444,89 +485,126 @@ def calibration(self):
             )
             return
 
-        self.start_of_calibration()
-        for current_epoch in range(self.config["calibration"]["max_epochs"]):
-            if self.check_epoch_conditions():
-                pass
-            else:
-                break
+        order_of_optimization = self.get_optimizers()
```

Comment:
should we now rename this to `optimizer_list` or similar?

Reply:
Perhaps `ordered_optimizers`?